README.md

```diff
@@ -7,13 +7,34 @@ Plugin that sets up a Python environment for building and running tests using
 Usage
 -----
 
-Apply the plugin to your project and configure the associated extension:
+Apply the plugin to your project:
```

Comment:
This looks like we should just refer you to the plugins website which tells you how: https://plugins.gradle.org/plugin/com.palantir.python.miniconda